examples/add_users_to_secure.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python
+#
+# Make sure all users are members of the Secure Operations team.
+#
+# As of when this script was written, there is only one team for
+# all Secure users. Newly-created users that land in the default
+# team for Monitor (such as those created via the API) will
+# therefore not be in the Secure Operations team. If you have an
+# environment where you want all users to have both Monitor and
+# Secure access by default, you could run this script periodically
+# (e.g. as a cron job) to make sure any such users are made part
+# of the Secure Operations team as well.
+#
+
+import os
+import sys
+import json
+import logging
+sys.path.insert(0, os.path.join(os.path.dirname(os.path.realpath(sys.argv[0])), '..'))
+from sdcclient import SdcClient
+
+#
+# Parse arguments
+#
+if len(sys.argv) != 2:
+    print 'usage: %s <sysdig-token>' % sys.argv[0]
+    print 'You can find your token at https://app.sysdigcloud.com/#/settings/user'
+    sys.exit(1)
+
+sdc_token = sys.argv[1]
+
+SECURE_TEAM_NAME = 'Secure Operations'
+
+#
+# As of when this script was written, the Secure Operations team does
+# not have the concepts of RBAC roles like "Read User" vs. "Edit User".
+# Rather, all members of the Secure team have full visibility within
+# Secure, which is associated with ROLE_TEAM_EDIT.
+#
+SECURE_TEAM_ROLE = 'ROLE_TEAM_EDIT'
+
+#
+# Instantiate the SDC client
+#
+sdclient = SdcClient(sdc_token, sdc_url='https://app.sysdigcloud.com')
+
+res = sdclient.get_teams(SECURE_TEAM_NAME)
+
+if res[0] == False:
+    print 'Unable to get teams: ', res[1]
+    sys.exit(1)
+memberships = {}
+for secure_team_user in res[1][0]['userRoles']:
+    memberships[secure_team_user['userId']] = secure_team_user['role']
+
+res = sdclient.get_users()
+
+if res[0] == False:
+    print 'Unable to get users: ', res[1]
+    sys.exit(1)
+all_users = res[1]
+
+#
+# The memberships passed into edit_team() are based on username
```

Comment:
@figarocorso Recently, during introduction of RBAC I have introduced pretty neat methods that you can use to manipulate memberships. Please refer to: `user_team_mgmt_extended.py`.

So basically, instead of fetching team and doing userName to id mapping you can just use this methods:
* list_memberships - returns (user-name, team-role) dictionary
* save_memberships - save only memberships instead of whole team object
* remove_memberships - remove memberships from team based on user names

This methods were introduced intentionally so end script user don't have to do userName->id mappings.

Please reach me if you need any assistance.